charts/dev/livebook/questions.yaml

```diff
@@ -16,11 +16,38 @@ questions:
                                     type: dict
                                     attrs:
                                       - variable: LIVEBOOK_PASSWORD
-                                        label: "LIVEBOOK_PASSWORD"
+                                        label: "Password (LIVEBOOK_PASSWORD)"
                                         description: "Password needed to access livebook (must be at least 12 characters)"
                                         schema:
                                           type: string
                                           default: "livebookiscool"
```

Comment:
Please dont set default password. user should explictly set one.

Also mark it as private, and required (if it is required)